05-classification.Rmd

```diff
@@ -47,16 +48,18 @@ prediction are called a **training set**. We call them a "training set" because
 we use these observations to train, or teach, our classifier so that we can use
```

Comment:
A few lines above (sorry GitHub won't let me comment on that line because you didn't change anything there), it says: "predicting a categorical class (sometimes called a *label*) for an observation given its other quantitative variables (sometimes called *features*)"

I don't think we want to say "quantitative variables" when we are generally talking about classification. Categorical predictors can be used with other models (e.g., logistic regression), just not k-nn. However, in this paragraph we are generally talking about classification, not just k-nn. So I think we should remove this, and then make this point about k-nn later on. 

I wonder, also, when we discuss the specifics of k-nn if we want to talk about how it is possible to use binary categorical predictors by coding them to 0 and 1? Students do this in the project a lot, and it would be nice to explain this in the book I think...

Reply:
Another point with the paragraph above, when we write:

"Generally, a classifier assigns an observation (e.g. a new patient) to a class (e.g.diseased or healthy) on the basis of how similar it is to other observations for which we know the class (e.g. previous patients with known diseases and symptoms). "

We don't specify that observation we are assigning to a class has no measured class/label. We know that, but a new learner might not from what we have written...

Reply:
agree